src/native/external/libunwind_extras/configure.cmake

```diff
@@ -13,13 +10,13 @@ if(CLR_CMAKE_HOST_WIN32)
     # Fake it until support is added
     check_include_files(stdalign.h HAVE_STDALIGN_H)
     if (NOT HAVE_STDALIGN_H)
-        configure_file(include/win/fakestdalign.h.in ${CMAKE_CURRENT_BINARY_DIR}/include/stdalign.h COPYONLY)
+        configure_file(${CLR_SRC_NATIVE_DIR}/external/libunwind/include/win/fakestdalign.h.in ${CMAKE_CURRENT_BINARY_DIR}/include/stdalign.h COPYONLY)
```

Comment:
This file is missing the licensing header.

Reply:
Same as above